azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/ClientEntity.java

```diff
@@ -58,6 +64,10 @@ protected final void setClosed() {
     }
 
     public final CompletableFuture<Void> close() {
+        if (TRACE_LOGGER.isInfoEnabled()) {
```

Comment:
move this below the sync() block - the logic in this class tries to de-duplicate multiple close calls to one :)